src/main/target/common_pre.h

```diff
@@ -348,6 +348,7 @@
 #define USE_ESC_SENSOR_INFO
 #define USE_CRSF_CMS_TELEMETRY
 #define USE_CRSF_LINK_STATISTICS
+#define USE_CRSF_V3
```

Comment:
This is likely to cause many legacy targets to stop building - this should really be under `#if (TARGET_FLASH_SIZE > 256)`.